### PR TITLE
ui: Rename `Strings needing action` to `Unfinished strings`

### LINKED ISSUE #1163: "strings needing action" doesn't actually count all strings needing action
Notably, the number does not include strings that are translated but have failing checks.

Also, for our purposes, we need to count strings with suggestions as "for review"; this will probably change if we implement a proper review workflow.

I propose counting strings with failing checks into "strings needing action".

Suggestions (and possibly comments) are trickier: the checklist item doesn't differentiate between untranslated and translated, which makes it impossible to get an exact number and wordcount of "strings needing work" for people who need or want that. OTOH, counting them in "strings needing action" would annoy people who don't care about suggestions on already translated strings.

At this point, i'm thinking of implementing a "Clean strings" checklist item, in addition to "Translated strings", and possibly a corresponding "Dirty strings" which is `total - clean`
##

### LINKED ISSUE #5896: "Untranslated/words" isn't actually untranslated
I can't distinguish untranslated strings with the new always-full bars.
This is compounded by not being able to sort for untranslated strings, or number of untranslated words.

https://hosted.weblate.org/languages/nb_NO/
bilde

The third category here is "Not translated strings" or somesuch, but aren't actually _missing_, as that would suggest.

For example: https://hosted.weblate.org/languages/nb_NO/asteroidos/#overview

### PR DESCRIPTION
**Pull request by nijel**
## Proposed changes

* This is a better description of the category.
* Consolidated labels for the filter and overview columns.

Fixes #1163
Fixes #5896

## Checklist

- [x] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added documentation to describe my feature.
- [x] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

**Comment by comradekingu** [comment]
Renaming strings needing work to "unfinished" isn't helping me find untranslated strings.
The combination of the two I don't understand, but "untranslated" it is not.
Calling only the progress bar category "unfinished" would at least be correct.
Sorting by untranslated (state:empty) first is what helps me find untranslated strings.
For the sake of simplicity, I supposed that is a compromise between my use-case and bulk categories.

Going so far as to rename the checkbox "needing action/work" everywhere, means now it seems like "unfinished" could mean partly translated, like translating 75% of a string and then coming back to finish it later.
Those strings actually make it in, so that is where I see a problem.

Taking something out of "unfinished" doesn't finish it.
It is so hard to get people to understand coverage as different from completion, and this will not help.

TL;DR Can only the places where "untranslated" (state:empty) and "needs work" are lumped together be called "unfinished", instead of this mass change?

**Comment by nijel** [comment]
This PR changes name for `state:<translated` strings to `Unfinished strings`, nothing else. It doesn't change anything else. Currently, these are sometimes called `Untranslated` and sometimes `Strings needing action`.

What you are looking for is `state:empty` and that will be covered at some point by https://github.com/WeblateOrg/weblate/issues/7008

**Comment by comradekingu** [comment]
It can be "empty" or "not translated" (as I think it originally was).
It seems you have a plan, and I maybe that coincides with mine or not.
I thought "strings needing action" was the checkbox, maybe it is, maybe it isn't.
